inference_segmentation.py

```diff
@@ -623,70 +549,56 @@ def main(params: dict) -> None:
     if valid_gpkg_set:
         logging.info('\nSuccessfully validated label data for benchmarking')
 
-    if task == 'classification':
-        classifier(params, list_img, model, device,
-                   working_folder)  # FIXME: why don't we load from checkpoint in classification?
+    gdf_ = []
+    gpkg_name_ = []
 
-    elif task == 'segmentation':
-        gdf_ = []
-        gpkg_name_ = []
+    # TODO: Add verifications?
+    if bucket:
+        bucket.download_file(loaded_checkpoint, "saved_model.pth.tar")  # TODO: is this still valid?
+        model, _ = load_from_checkpoint("saved_model.pth.tar", model)
+    else:
+        model, _ = load_from_checkpoint(loaded_checkpoint, model)
 
-        # TODO: Add verifications?
+    # Save tracking TODO put option not just mlflow
+    if 'tracker_uri' in locals() and 'run_name' in locals():
```

Comment:
No major change here, git's just having a hard time following the movement of big blocks of code.